# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/ActorsParent.cpp
# Commit: fa95640baa86
# Full Hash: fa95640baa86a86cb33f6bf9985f3a771c15e9f4
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-08-21 16:47:43
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Added missing assignment of Exists result to rv. r=ttung,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D41191
# ==============================================================================

diff -r f0f5d12cc2b7 -r fa95640baa86 dom/indexedDB/ActorsParent.cpp
--- a/dom/indexedDB/ActorsParent.cpp	Wed Aug 21 09:27:31 2019 +0200
+++ b/dom/indexedDB/ActorsParent.cpp	Wed Aug 21 09:27:46 2019 +0200
@@ -20034,7 +20034,7 @@
     return rv;
   }
 
-  markerFile->Exists(&exists);
+  rv = markerFile->Exists(&exists);
   if (NS_WARN_IF(NS_FAILED(rv))) {
     return rv;
   }
